### PR TITLE
69 Create menu

### PR DESCRIPTION
**Pull request by jema28**
**Issue**: #69

### CODE REVIEW

File: src/components/common/WorkCard/index.js [hunk 1]
```diff
@@ -1,74 +1,28 @@
...
-      break
-    case 3:
-      light = topDestance > 3426
-      break
-    case 4:
-      light = topDestance > 3730
-      break
-    default:
-      break
-  }
```
Comment (yosefanajjar) [inline src/components/common/WorkCard/index.js:35]: this code was handling the brightness of the work card is this effect not needed anymore?

File: src/components/layout/Footer/index.js [hunk 2]
```diff
@@ -1,73 +1,25 @@
...
+          target="_top"
+        >
+          hello@yallacooperative.com
+        </EmailLink>
+      </ContentWrapper>
+    </FooterWrapper>
+    <Copyright>
+      <p>© 2020 Yalla Cooperative</p>
+    </Copyright>
+  </>
```
Comment (yosefanajjar) [inline src/components/layout/Footer/index.js:22]: is this how the footer looks now? the design changed?

File: src/components/common/WorkCard/index.js (hunk 1 above)
Comment (jema28) [reply src/components/common/WorkCard/index.js:35]: I think it's not needed at the moment as we have two case studies so was planning on putting it back when we have four

File: src/components/layout/Footer/index.js (hunk 2 above)
Comment (jema28) [reply src/components/layout/Footer/index.js:22]: For now yeh as we don't have proper content for the addresses etc and the different links but once everything's done planning to change it back to the original

File: src/components/layout/Header/Nav.js [hunk 3]
```diff
@@ -4,29 +4,52 @@ import styled from 'styled-components'
 import PropTypes from 'prop-types'
 import Logo from '../../../../static/svg/yalla-logo.svg'
 import LogoBlack from '../../../../static/svg/yall-logo-black.svg'
-import ContentWrapper from '../../contentWrapper'
+import { Wrapper as ContentWrapper } from '../../contentWrapper'
+import { breakpoints, spacings } from '../../../utils'
 
-const MenuContainer = styled.div`
+const MenuContainer = styled(Link)`
```
Comment (yosefanajjar) [inline src/components/layout/Header/Nav.js:10]: the name is confusing if it's a link then change the name

**Review by yosefanajjar — APPROVED** [review]
looks fine to me just didn't expect to find changes in other components in the Nav pull request

**Comment by yosefanajjar** [comment]
image

this needs a fix

**Comment by jema28** [comment]
Ah yeh, I needed some of the changes from the landing page so that's why it's all in here @yosefanajjar

**Comment by jema28** [comment]
> image
> 
> this needs a fix

Where was this sorry @yosefanajjar? What needs fixing?

**Comment by yosefanajjar** [comment]
@jema28 in the landing I was viewing it on a 24inch monitor

**Pull request merged**